modules/installation-aws-cluster-network-mtu.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-aws-localzone.adoc (Installing a cluster on AWS with worker nodes on AWS Local Zones)
+// * installing/installing-aws-wavelength-zone.adoc (Installing a cluster on AWS with worker nodes on AWS Wavelength Zones)
+// * installing/installing_aws/installing-aws-outposts-remote-workers.adoc[Installing a cluster on AWS with remote workers on AWS Outposts]
+
+ifeval::["{context}" == "installing-aws-localzone"]
+:local-zone:
+endif::[]
+ifeval::["{context}" == "installing-aws-wavelength-zone"]
+:wavelength-zone:
+endif::[]
+ifeval::["{context}" == "installing-aws-outposts-remote-worker"]
+:aws-outposts:
+endif::[]
+
+:_mod-docs-content-type: CONCEPT
+[id="installation-aws-cluster-network-mtu_{context}"]
+= Customizing Cluster Network MTU
+
+Before you deploy a cluster on AWS, you can customize the Cluster Network (maximum transmission unit) MTU to meet your infrastructure's needs. 
+
+By default, on installing a cluster with supported {zone-type} capabilities, the Cluster Network MTU gets automatically adjusted to the lower value that the network plugin accepts.
+
+[IMPORTANT]
+====
+Setting an unsupported MTU value for ECS instances that operate in the {zone-type} environment can cause performance issues for your {product-title} cluster.
+====
+
+ifdef::local-zone[]
+If the Local Zone supports higher MTU in between EC2 instances in the Local Zone within a Region, you might need to manually configure the higher value to increase the network performance of the cluster network.
+endif::local-zone[]
+
+ifdef::wavelength-zone[]
+If the Wavelength Zone supports higher MTU in between EC2 instances in the Wavelength Zone within a Region, you might need to manually configure the higher value to increase the network performance of the cluster network.
+endif::wavelength-zone[]
+
+ifdef::aws-outposts[]
+If AWS Outposts supports higher MTU in between EC2 instances in an AWS Outpost within a Region, you might need to manually configure the higher value to increase the network performance of the cluster network. 
+endif::aws-outposts[]
+
+You can customize the MTU for a cluster by specifying the `networking.clusterNetworkMTU` parameter in the `install-config.yaml` configuration file.
```

Comment:
How about?

> All zone subnets in {zone-type} must support the higher MTU value, so that each node in a subnet zone can succesfully deploy your workloads.

Reply:
> All subnets in {zone-type} must support the higher MTU value, so that each node in that zone can successfully communicate with services in the Region and deploy your workloads.

Reply:
Good suggestion. Updated.